sda/cmd/api/api.md

```diff
@@ -57,6 +57,22 @@ Admin endpoints are only available to a set of whitelisted users specified in th
     curl -H "Authorization: Bearer $token" -H "Content-Type: application/json" -X POST -d '{"accession_id": "my-id-01", "filepath": "/uploads/file.c4gh", "user": "testuser"}' https://HOSTNAME/file/accession
     ```
 
+- `/file/verify/*accession`
+  - accepts `PUT` requests with an accession ID as the last element in the query
+  - triggers re-verification of the file with the specific accession ID.
+
+  - Error codes
+    - `200` Query execute ok.
+    - `404` Error due to non existing accession ID.
+    - `401` Token user is not in the list of admins.
+    - `500` Internal error due to DB or MQ failures.
+
+    Example:
+
+    ```bash
+    curl -H "Authorization: Bearer $token" -H "Content-Type: application/json" -X POST -d '{"accession_id": "my-id-01", "filepath": "/uploads/file.c4gh", "user": "testuser"}' https://HOSTNAME/file/accession
```

Comment:
The endpoint accepts `PUT` request, but the example is provided with a `POST` request.